R/effectsize.R

```diff
@@ -54,6 +54,8 @@
 #'
 #' ## Bayesian Hypothesis Testing
 #' ## ---------------------------
+#' \donttest{
+#' \dontrun{
```

Comment:
dontrun should be enough. Is just timing is crucial, donttest should be enough. donttest is not needed when using dontrun.

Reply:
I never which if these is enough...